### ISSUE TITLE
can't compile RAS from source

### ISSUE THREAD

**Issue by pete111** [dillbyrne/random-agent-spoofer#366]
Hello, 
I downloaded RAS and I am trying co compile it from original source. This python error occurs:

(C:\addon-sdk-1.17) C:\addon-sdk-1.17\random_agent_spoofer>cfx
Traceback (most recent call last):
  File "C:\addon-sdk-1.17\bin\cfx", line 33, in <module>
    cuddlefish.run()
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\__init__.py", line 753, in run
    abort_on_missing=options.abort_on_missing)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 669, in build_mani
    mxt.build(scan_tests, test_filter_re)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 201, in build
    top_me = self.process_module(top_mi)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 411, in process_mo
    them_me = self.find_req_for(mi, reqname, looked_in, locations)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 479, in find_req_f
    looked_in)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 568, in _get_modul
ackage
    return self._handle_module(mi)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 560, in _handle_mo
    self.process_module(mi)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 411, in process_mo
    them_me = self.find_req_for(mi, reqname, looked_in, locations)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 479, in find_req_f
    looked_in)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 568, in _get_modul
ackage
    return self._handle_module(mi)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 560, in _handle_mo
    self.process_module(mi)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 411, in process_mo
    them_me = self.find_req_for(mi, reqname, looked_in, locations)
  File "C:\addon-sdk-1.17\python-lib\cuddlefish\manifest.py", line 471, in find_req_f
    raise BAD("too many ..")
cuddlefish.manifest.BadModuleIdentifier: too many .. in require(../package.json) from ModuleInfo [random-agent-spoofer lib Panel] (C:\addon-sdk-1.17\random_agent_spoofer\lib\Panel.js, None)

What's wrong with it?

**Comment by dillbyrne** [comment]
@pete111 cfx is the old python tool. SDK based addons have moved to jpm now. See https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/Getting_Started_%28jpm%29

TL;DR
setup jpm
clone RAS
cd RAS
jpm xpi

**Comment by dillbyrne** [comment]
I'm closing this now. Feel free to reopen it if you are still having trouble. Note to run your own unsigned xpi you will need to use the dev edition and toggle the preference for signatures required to false
